index.js

```diff
@@ -36,6 +36,15 @@
             context.map.center = new L.LatLng(center[1], center[0]);
             context.map.zoom = center[2];
         },
+        bounds: function(context, b) {
+            // TileJson order is lng lat lng lat
+            // left, bottom, right, top
+            var left = b[0], bottom = b[1], right = b[2], top = b[3];
+            context.bounds = L.latLngBounds([bottom, left], [top, right]);
```

Comment:
This looks correct, `L.LatLngBounds` should be used here, but from the example we would pass projected coordinates in here, if I understand correctly, which would not be correct.

Leaflet's CRS object now has a [`getProjectedBounds`](http://leafletjs.com/reference-1.0.3.html#crs-getprojectedbounds) method which should return the projected bounds if specified - I don't remember if Proj4Leaflet supports that, though.